scripts/sentiment_analysis/finetune_lm.py

```diff
@@ -333,11 +335,12 @@ def fit_batch(self, estimator, train_batch, batch_axis=0):
 
 
 class ClipGradientHandler(GradientUpdateHandler):
+    """handler for clipping gradient"""
     def __init__(self, clip=None, **kwargs):
         super().__init__(**kwargs)
         self.clip = clip
 
-    def batch_end(self, estimator, *args, **kwargs):
+    def batch_end(self, estimator, *argss, **kwargs):
```

Comment:
typo?

Reply:
Due to redefining name 'args' from outer scope (line 123) (redefined-outer-name), I renamed args to argss

Reply:
I see. Thank you for clarifying. One alternative is to move the problematic outer scope statements to the end of the file, inside the if statement.
In general it's bad practice to do any "actual computation" outside of the if statement at the end of the file.

Reply:
OK, I get it. By the way, can I disable the linting issue(W0621)?